pubspec.lock

```diff
@@ -545,18 +545,18 @@ packages:
     dependency: transitive
```

Comment:
Should the `pubspec.lock` file be removed from git tracking entirely?

Reply:
I generally don't check these in. You don't want to check them in for a library; the guidance is to check them in for an app.

I would check them in for a high-volume app where you wanted reproducible control over the exact bits you ship. For something like this app however, I'd be fine w/ letting the exact resolved versions float.